__init__.py

```diff
@@ -14,9 +14,12 @@
 from bpy.app.handlers import persistent
 
 import os
+import sys
 import json
 import importlib
 
+#a little hacky bs
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Just kinda curious, what does this do?

Reply:
it was to solve an issue where python was having issues resolving relative imports. I found out about this solution here:
https://itsmycode.com/importerror-attempted-relative-import-with-no-known-parent-package/

Reply:
Oh cool, interesting!